CHANGES.rst

```diff
@@ -28,6 +28,8 @@
 - Reintroduce optional support for ``subunit``, now with support for both
   version 1 and version 2 of its protocol.
 
+- Added support for Python 3.7.
```

Comment:
Maybe move this next to the 'Dropped support for Python 3.3' line?  Or, alternatively, move that line down.

(I think it makes sense to keep supported Python version notes next to each other in the changelog.)

Reply:
Yes, you're right. I'll do that from now on. 
Thank you for fixing it!